coursedata/level-defaults/nb_NO.yaml

```diff
@@ -1,354 +1,354 @@
-        example: name = 'Hedy'
-        demo_code: "name = 'Hedy the Robot'\n{print} 'Hello ' name\n"
-    -   name: quotes after {if} comparison
-        explanation: Text with quotation marks after {if}
-        example: name = 'Hedy'
-        demo_code: "name = {ask} 'Who are you?'\n{if} name = 'Hedy'\n    {print} 'Hi there!'\n"
-    -   name: quotes in list
-        explanation: A list with quotation marks
-        example: name = 'Hedy', 'Felienne'
-        demo_code: "superheroes = 'Iron Man', 'Batman', 'Superman'\n{print} superheroes {at} {random}\n"
-    start_code: "{print} 'decimal numbers now need to use a dot'\n{print} 2.5 + 2.5"
-    intro_text: "## Numbers and texts\n**Decimal numbers**\nSo far, Hedy did not allow for decimal numbers like 1.5, but now we do allow that. Note that computers use the `.` for decimal numbers.\n"
-    example_code: "## Example Hedy code\n```\n{print} 'Two and a half plus two and a half is...'\n{print} 2.5 + 2.5\n```\n"
-    intro_text_2: "**All texts need to be in quotation marks**\nFor this level on you will also have to use quotation marks when storing a text with `=`:\n"
-    example_code_2: "```\nname = 'Hedy the Robot'\n{print} 'Hello ' name\n```\n"
-    intro_text_3: "**All items in lists need quotation marks too**\nLists are texts, so they need quotation marks too. Mind that each single item on the list has quotation marks.\nThis allows you to save two words as 1 item on the list, for example 'Iron Man'.\n"
-    example_code_3: "```\nsuperheroes = 'Spiderman', 'Batman', 'Iron Man'\n{print} superheroes {at} {random}\n```\n"
-    intro_text_4: "**All text after `{if}` comparisons need quotation marks too**\n"
-    example_code_4: "```\nname = {ask} 'What is your name?'\n{if} name = 'Hedy the Robot'\n    {print} 'Hi there!'\n```\n"
-    intro_text_5: "**Numbers don't need quotation marks**\nFor numbers, you do not use quotation marks in the `=`:\n"
-    example_code_5: "```\nscore = 25\n{print} 'You got ' score\n```\n"
-    intro_text_6: "**Maths with words** In this level it is also allowed to use the + sign with texts. For example:\n"
-    example_code_6: "```\na = 'Hello '\nb = 'world!'\nprint a + b\n```\n"
+        demo_code: "{print} 'Kjør på med kalkulasjoner!'\n{print} 'To og en halv pluss to og en halv er lik...'\n{print} 2.5 + 2.5\n"
+    -   name: tildel tekst
+        explanation: Tekst med fnutter etter `=`
+        example: navn = 'Hedy'
+        demo_code: "navn = 'Roboten Hedy'\n{print} 'Hello ' navn\n"
+    -   name: apostrof etter {if}sammenligning
+        explanation: Tekst med fnutter etter {if}
+        example: navn = 'Hedy'
+        demo_code: "navn = {ask} 'Hvem er du?'\n{if} navn = 'Hedy'\n    {print} 'Hei hei!'\n"
+    -   name: hermetegn i liste
+        explanation: En liste med fnutter
+        example: navn = 'Hedy', 'Felienne'
+        demo_code: "superhelter = 'Iron Man', 'Batman', 'Superman'\n{print} superheroes {at} {random}\n"
```

Comment:
This doesn't work! As `superhelter` is a variable it should be translated everywhere in the program or not. The tests currently crash because `superheroes at random` can't find the variable `superheroes`.

Reply:
Fixed this in Weblate!